test/commands/common/issue-update.test.ts

```diff
@@ -0,0 +1,286 @@
+import nock from "nock";
+import myProbotApp from "../../../src";
+import { Probot } from "probot";
+import {
+  updateNotification,
+  updateStatus,
+} from "../../../src/commands/common/issue-update";
+
+import payload from "../../fixtures/issues.ping.comment.json";
+import typeorm = require("typeorm");
+
+const fs = require("fs");
+const path = require("path");
+
+const issueBodyTemplate = `
+## Description
+
+这是一段背景
+
+## Tasks
+
+- 任务1
+- 任务2
+
+## Score
+
+- 100
+
+## Mentor
+
+- @rust-in
+
+## Recommended Skills
+- Go
+- MarkDown
+
+## Learning Materials(optional)`;
+
+const notificationHead = `
+
+
+## :warning:Notification:warning:
+
+pong! I am challenge bot.<!-- probot:Notification -->
+
+`;
+
+describe("Issue Update", () => {
+  let probot: any;
+  let mockCert: string;
+
+  beforeAll((done: Function) => {
+    fs.readFile(
+      path.join(__dirname, "../../fixtures/mock-cert.pem"),
+      (err: Error, cert: string) => {
+        if (err) return done(err);
+        mockCert = cert;
+        done();
+      }
+    );
+  });
+
+  beforeEach(() => {
+    // @ts-ignore
+    typeorm.createConnection = jest.fn().mockResolvedValue(null);
+    nock.disableNetConnect();
+    probot = new Probot({ id: 123, cert: mockCert });
+    // Load our app into probot
+    probot.load(myProbotApp);
+
+    nock("https://api.github.com")
+      .post("/app/installations/2/access_tokens")
+      .reply(200, {
+        token: "test",
+        permissions: {
+          issues: "write",
+        },
+      });
+  });
+
+  test("updateNotification function test", () => {
+    const case1: [string, string, string, string][] = [
```

Comment:
Why use two-dimensional arrays? Actually, the array `case1` only has one element. And maybe you can use more descriptive variable name, that may improve the readability of your code.